chrome/assets_forge/api-priv-chrome.js

```diff
@@ -129,7 +129,14 @@ forge.is.desktop = function () {
  * @return {boolean}
  */
 forge.is.chrome = function () {
-	return true;
+    return navigator.userAgent.toLowerCase().indexOf('chrome') > -1;
```

Comment:
We should keep using hardcoded flag, the userAgent can be spoofed and we shouldn't rely on it. If we use the same code for Firefox and Chrome BE (btw. what about Edge, isn't it also Chrome now?) then we should flip the flags via build script or environment variable during build

Reply:
Hi Ivo, currently, Edge is still checked by userAgent. 
I will look into build script. 
-- Ngoc Nguyen --